client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.html

```diff
@@ -164,13 +165,48 @@ <h2>{{ 'Participants' | translate }}</h2>
     <ng-template matMenuContent let-user="user">
         <!-- Presence -->
         <mat-checkbox
+            *ngIf="viewport.isMobileSubject | async"
             class="mat-menu-item"
             (change)="setPresent(user)"
             [checked]="isUserPresent(user)"
             [disabled]="isPresentToggleDisabled(user)"
         >
             <span>{{ 'Present' | translate }}</span>
         </mat-checkbox>
+
+        <button *osPerms="permission.userCanManage" mat-menu-item (click)="openEditInfo(user, $event)">
+            <mat-icon>edit</mat-icon>
+            <span>{{ 'Edit details' | translate }}</span>
+        </button>
```

Comment:
After clicking "Edit details" the menu should closed - as in agenda 3-dot-menu (with "edit details").